hatchways/team-callback-cats
# s3 initial setup, config and upload route

### What this PR does (required):
- Create "lovingdogsitter" aws S3 bucket and set up environment variables in .env
- Obtain bucket URL & create upload route function

### Screenshots / Videos (required):
Postman test
-![Screen Shot 2021-08-16 at 9 44 48 PM](https://user-images.githubusercontent.com/63027785/129665013-ea33c6dd-4fd3-456b-a3d7-f1d4ee9fbf62.png)
server/routes/upload.js
![Screen Shot 2021-08-16 at 9 45 28 PM](https://user-images.githubusercontent.com/63027785/129665057-a150fd54-0bbd-474f-a5ca-07c3232fa0b1.png)

### Any information needed to test this feature (required):
- See AWS keys in Slack channel
- run **npm install** to update dependencies list. I added multer, multer-s3, aws-sdk

### Any issues with the current functionality (optional):
- Still need to create routes and variables for different requests, and connect to front end
